data/brands/amenity/pharmacy.json

```diff
@@ -4004,21 +4004,28 @@
     },
     {
       "displayName": "サツドラ",
-      "id": "sapporodrugstore-650eee",
+      "id": "satudora-650eee",
       "locationSet": {"include": ["jp"]},
       "tags": {
         "alt_name": "サッポロドラッグストアー",
-        "alt_name:en": "Satsudora",
+        "alt_name:en": "Sapporo Drug Store",
         "alt_name:ja": "サッポロドラッグストアー",
+        "alt_name:zh": "札幌薬粧",
         "amenity": "pharmacy",
         "brand": "サツドラ",
-        "brand:en": "Sapporo Drug Store",
+        "brand:en": "SATUDORA",
         "brand:ja": "サツドラ",
+        "brand:th": "ซัทสึโดระ",
         "brand:wikidata": "Q11304804",
+        "brand:zh-Hans": "札幌药妆",
+        "brand:zh-Hant": "札幌藥妝",
         "healthcare": "pharmacy",
         "name": "サツドラ",
-        "name:en": "Sapporo Drug Store",
-        "name:ja": "サツドラ"
+        "name:en": "Satudora",
+        "name:ja": "サツドラ",
+        "name:th": "ซัทสึโดระ",
```

Comment:
Curious why you need to add Thai and Chinese here

Reply:
This is because the official website has pages in these languages as tax free stores and I thought it would be useful to make them multilingual.
But I will remove if redundant.
https://satudora.jp/taxFree/zh-cn/

Reply:
I tend to remove because they are not commonly used languages ​​in the whole of Japan, like we don't need to add dozens of languages ​​​​to KFC at the same time, and I've done this.

Reply:
I see, thank you for removing.